File: app/Dto/Bill.php
<?php

namespace App\Dto;

use Ramsey\Uuid\UuidInterface;

class Bill
{
    public function __construct(
        public readonly UuidInterface $debtId,
        public readonly String $name,
        public readonly String $email,
        public readonly float $value,
        public readonly String $dueDate,
        public readonly String $governmentId,
        public readonly String $pathPdf
    )
    {
    }
}

File: src/main/kotlin/application/domain/models/Bill.kt
package application.domain.models

import java.util.UUID

data class Bill(
    val name: String,
    val email: String,
    val value: Double,
    val debtId: UUID,
    val dueDate: String,
    val governmentId: String,
) {
    companion object {

        fun from(line: Map<String, String>): Bill {
            return Bill(
                name = line.getValue("name"),
                email = line.getValue("email"),
                value = line.getValue("debtAmount").toDouble(),
                debtId = UUID.fromString(line.getValue("debtId")),
                dueDate = line.getValue("debtDueDate"),
                governmentId = line.getValue("governmentId"),
            )
        }
    }
}
